frontend/src/app/organization/organization-contact/organization-contact.component.ts

```diff
@@ -0,0 +1,30 @@
+import {Component, EventEmitter, Input, OnInit, Output, ViewChild} from '@angular/core';
```

Comment:
we leave empty space between `{`/`}` and symbols that are imported, like `import { NgForm } from '@angular/forms';`